OpenRA.Mods.Common/Activities/GenericDockSequence.cs

```diff
@@ -191,20 +187,20 @@ public virtual void PlayUndockClientAnimation(Actor self, Action after)
 		void NotifyDocked(Actor self)
 		{
 			foreach (var nd in notifyDockClients)
-				nd.Docked(self, DockHostActor);
+				nd.Linked(self, DockHostActor);
 
 			foreach (var nd in notifyDockHosts)
-				nd.Docked(DockHostActor, self);
+				nd.Linked(DockHostActor, self);
 		}
 
 		void NotifyUndocked(Actor self)
 		{
 			foreach (var nd in notifyDockClients)
-				nd.Undocked(self, DockHostActor);
+				nd.Unlinked(self, DockHostActor);
 
-			if (DockHostActor.IsInWorld && !DockHostActor.IsDead)
```

Comment:
Why remove the is in world check?

Reply:
the commit message says

> We should notify host even when it's not in world

As in, it should be the job of the host to decide what to do with a notification